CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## Release History
+### 1.9.0-beta.1 (Unreleased)
+#### Key Bug Fixes
+* Fixed issue in `CosmosDBSourceConnector` where no record being read when configured `connect.cosmos.offset.useLatest` to be false. [PR 516](https://github.com/microsoft/kafka-connect-cosmosdb/pull/516)
```

Comment:
We want to call this out as a breaking change, no?

Reply:
good point Kushagra, maybe I can add some tests around no lease documents will be updated

Reply:
Good catch Kushagra, updated

Reply:
updated